packages/editor/src/plugins/rows/editor.tsx

```diff
@@ -18,7 +18,7 @@ export function RowsEditor({ state, config, id }: RowsProps) {
   function insertRowWithSuggestionsOpen(insertIndex: number) {
```

Comment:
This function's name is outdated now. It only made sense when it added a Text plugin with `/`, where the `/` would trigger a suggestions list to show.

Reply:
Renamed